great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -148,6 +158,179 @@ def get_batch_id(
 
         return batch_ids[0]
 
+    def get_numeric_metric(
+        self,
+        batch_id: str,
+        validator: Validator,
+        metric_name: str,
+        metric_domain_kwargs: Optional[Union[str, dict]] = None,
+        metric_value_kwargs: Optional[Union[str, dict]] = None,
+        enforce_numeric_metric: Optional[Union[str, bool]] = False,
+        fill_nan_with_zero: Optional[Union[str, bool]] = False,
+        domain: Optional[Domain] = None,
+        variables: Optional[ParameterContainer] = None,
+        parameters: Optional[Dict[str, ParameterContainer]] = None,
+    ) -> Dict[str, Union[Number, Dict[str, Any]]]:
+        metric_domain_kwargs = build_metric_domain_kwargs(
+            batch_id=batch_id,
+            metric_domain_kwargs=metric_domain_kwargs,
+            domain=domain,
+            variables=variables,
+            parameters=parameters,
+        )
+        # Obtain value kwargs from rule state (i.e., variables and parameters); from instance variable otherwise.
+        metric_value_kwargs = get_parameter_value_and_validate_return_type(
+            domain=domain,
+            parameter_reference=metric_value_kwargs,
+            expected_return_type=None,
+            variables=variables,
+            parameters=parameters,
+        )
+
+        # Obtain enforce_numeric_metric from rule state (i.e., variables and parameters); from instance variable otherwise.
+        enforce_numeric_metric = get_parameter_value_and_validate_return_type(
+            domain=domain,
+            parameter_reference=enforce_numeric_metric,
+            expected_return_type=bool,
+            variables=variables,
+            parameters=parameters,
+        )
+
+        # Obtain fill_nan_with_zero from rule state (i.e., variables and parameters); from instance variable otherwise.
+        fill_nan_with_zero = get_parameter_value_and_validate_return_type(
+            domain=domain,
+            parameter_reference=fill_nan_with_zero,
+            expected_return_type=bool,
+            variables=variables,
+            parameters=parameters,
+        )
+
+        metric_configuration_arguments: Dict[str, Any] = {
+            "metric_name": metric_name,
+            "metric_domain_kwargs": metric_domain_kwargs,
+            "metric_value_kwargs": metric_value_kwargs,
+            "metric_dependencies": None,
+        }
+        metric_value: Number = validator.get_metric(
+            metric=MetricConfiguration(**metric_configuration_arguments)
+        )
+        if enforce_numeric_metric:
```

Comment:
Could this and L224 combine their conditionals into a single line? Very much a stylistic preference without impact on code so I wouldn't dwell on it if you want to merge.

`i.e. if enforce_numeric_metric and is not is_numeric:`

Reply:
@cdkini I prefer to leave it as is, because `enforce_numeric_metric` is common (hence, should be factored out and not evaluated twice, regardless of how inexpensively that would be).